src/index.ts

```diff
@@ -148,6 +154,59 @@ const moveTo = (a: Set<string>, b: Set<string>, key: string): string => {
 
 const isExample = (filePath: string): boolean => filePath.split(path.sep).some(name => name === 'examples')
 
+const isFolderContainsReadme = async (folder: string): Promise<boolean> => {
+  const readmePath = path.resolve(folder, 'readme.md')
+  return fs.exists(readmePath)
+}
+
+/**
+ * Validate each RP folder must have its readme file.
+ *
+ * @param specification specification folder
+ */
+const validateRPFolderMustContainReadme = (specification: string): asyncIt.AsyncIterableEx<err.Error> =>
```

Comment:
Can we pass in the PR file path to do validation instead of the root path of specification?

Reply:
I think both are same.

Reply:
As synced offline, leave this as is to keep same root level search as other rules. And will consider to base on changed file list of PR for all rules in future refactoring.